tests/plan/import/modify-data/plans.fmf

```diff
@@ -1,16 +1,53 @@
-/must-be-imported-and-modified:
-    summary: Test of correct environment variable expansion of ${VARIABLE}
+/imported:
+    summary: Test of correct environment variable expansion
     provision:
         how: local
     execute:
-        script: test "${VARIABLE}" == "foobar"
+        # Cannot use `$VARIABLE`, tmt would overwrite it with the value from `environment` above.
```

Comment:
Is this still applicable since now the environment is defined separately in `/imported/bare` and `/imported/with-environment`? Also it's not defined "above".

Reply:
I believe it still applies because the `with-environment` plan would inherit this test and get bitten by the premature interpolation. But I will fix the "above" part, which can be rephrased.